.github/workflows/snyk-scan.yml

```diff
@@ -20,17 +19,17 @@ jobs:
           fetch-depth: 0 # To fetch all commits history on branch (Refer: https://github.com/tj-actions/changed-files#usage)
        
       - name: Check changed Deps files
-        uses: tj-actions/changed-files@v10
+        uses: tj-actions/changed-files@v35
         id: changed-files
         with:
           files: | # This will match all the files with below patterns
-            build.gradle$
-            requirements.txt$
-            package.json$
+            *build.gradle
```

Comment:
why we need *

Reply:
It is the new way of matching patterns for file names. `*` is used instead of `$`. https://github.com/marketplace/actions/changed-files
![image](https://user-images.githubusercontent.com/67860730/218741185-201fd27f-6674-4415-b580-6720a6ffeaa7.png)

Reply:
ok got it